docs/api/tutorials/start.md

```diff
@@ -0,0 +1,206 @@
+
+```perl6
+my $*RED-DB = database “SQLite”, database => 'test.sqlite3'; # Now a file `test.sqlite3` will be created
+```
+
+Next, we need to create a table itself:
+
+
+```perl6
+Person.^create-table;
+```
+
+Methods marked with `^` are called "meta methods" and are used in Red
+for all kinds of operations on models. In this case, calling `^create-table`
+creates a table with name `person`.
+
+### Insertion of new records
+
+Let's insert a new record into it. In SQL it could be:
+
+```sql
+INSERT INTO person (name) VALUES 'John';
+```
+
+In Red we can express it this way:
+
+```perl6
+my $person = Person.^create(name => 'John');
+```
+
+We call the `^create` method on type object `Person` and assign the result
+to the `$person` variable. The assignment is not necessary:
+
+```perl6
+Person.^create(name => 'John');
+```
+
+The `^create` method returns the created object to work with, though
+this result can be simply ignored.
+
+The `$.id` attribute is auto-generated and there is no need to specify it,
+while `$.name` attribute must not be null, so we have to specify it:
+
+```perl6
+Person.^create; # error
+```
+
+### Update of records
+
+Let's try to update our record. In SQL it could be:
+
+```sql
+UPDATE person SET name = ‘John Doe’ WHERE id = 1;
+```
+
+To do the same in Red, we use setters and a call to `^save`:
+
+```perl6
+$person.name = 'John Doe';
+$person.^save;
+```
+
+All changes to an object that represents the record are lazy,
+which means the database connection is not used until the `^save`
+method is called.
+
+The method `^save` is useful not only for UPDATE operation, but it can be used on
+INSERT too:
+
+```perl6
+my $person2 = Person.new(name => 'Joan'); # ^create is not used
+$person2.^save; # does INSERT, not UPDATE
+```
+
+### Selecting records
+
+Lets add some more records:
+
+```perl6
+Person.new(name => "Paul"); # Method call with parentheses and an arrow pair
+Person.new: :name<Miki>;    # Semicolon form of method call is used
+```
+
+Lets begin with selecting all records of the table:
+
+```sql
+SELECT * FROM person;
+```
+
+In Red, `^all` method is used:
+
+```perl6
+for Person.^all -> $person { say $person }
+```
+
+Method `^all` returns an instance of class `Seq` that is a lazy sequence of records returned.
+
+```sql
+SELECT * FROM person WHERE person.name like 'Jo%';
+```
+
+The query above selects all records where name starts with 'Jo'. In Red, you can use Perl 6 `grep`
+method to specify clauses of the select query:
+
+```perl6
+for Person.^all.grep(*.name.starts-with('Jo')) -> $person { say $person }
+```
+
+Note that this call chain will result into an equivalent of the SQL code above,
+filtering values happens at the database level, not at Perl 6 level.
+
+```sql
+SELECT * FROM person WHERE person.name like 'Jo%' AND person.id = 2;
+```
+
+To express the query above, calls to `grep` can be combined:
+
+```perl6
+for Person.^all.grep(*.name.starts-with('Jo')).grep(*.id == 2) -> $person { say $person }
```

Comment:
You could use
```perl6
Person.^all.grep({ .name.starts-with('Jo') && .id == 2 })
```

as well

Reply:
I also tried `.grep(*.name.starts-with('Jo') && *.id == 2)`, but it transformed only the left side of the condition, is it a known bug?

Reply:
@Altai-man, no... that would generate a block that would receive 2 args... that’s the problem...